Packs/Threat_Crowd/Integrations/ThreatCrowdV2/ThreatCrowd_v2.yml

```diff
@@ -0,0 +1,391 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: TheatCrowdv2
+  version: -1
+configuration:
```

Comment:
i think we are lacking the extended_data param.
pleas conform to the standard presented in VT v3.